pkg/osbuild/device.go

```diff
@@ -206,12 +206,21 @@ func getDevices(path []disk.Entity, filename string, lockLoopback bool) (map[str
 	return do, parent
 }
 
-func pathdot(path string) string {
-	if path == "/" {
-		return "root"
+// pathEscape implements similar path escaping as used by systemd-escape
+// https://github.com/systemd/systemd/blob/main/src/basic/unit-name.c#L389
```

Comment:
NITPICK: Let's get the permalink for this so it doesn't link to a different line when the file changes in `main`.

Reply:
fixed